src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsLagrangianFEM.cpp

```diff
@@ -146,20 +146,17 @@ void SolidMechanicsLagrangianFEM::registerDataOnMesh( Group & meshBodies )
     nodes.registerField< solidMechanics::incrementalDisplacement >( getName() ).
       reference().resizeDimension< 1 >( 3 );
 
-    if( m_timeIntegrationOption != TimeIntegrationOption::QuasiStatic )
-    {
-      nodes.registerField< solidMechanics::velocity >( getName() ).
-        reference().resizeDimension< 1 >( 3 );
+    nodes.registerField< solidMechanics::velocity >( getName() ).
```

Comment:
can we maybe add a flag to only register these fields when the chombo coupling is on? coz registering them when they are not needing goes kind of in the opposite direction of reducing our memory footprint as much as possible.

Reply:
Yeah I like that as a solution better.